helm/exporter-node/templates/daemonset.yaml

```diff
@@ -21,14 +21,17 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: "{{ .Values.image.pullPolicy }}"
           args:
-          {{ if lt .Values.image.tag "v0.15.0" }}
+          {{- if lt .Values.image.tag "v0.15.0" }}
             - --collector.procfs=/host/proc
             - --collector.sysfs=/host/sys
-          {{ else }}
+          {{- else }}
             - --path.procfs=/host/proc
             - --path.sysfs=/host/sys
-          {{ end }}
+          {{- end }}
             - --web.listen-address=0.0.0.0:{{ .Values.service.containerPort }}
+          {{- if .Values.extraVolumeMounts }}
+{{ toYaml .Values.extraArgs | indent 12 }}
```

Comment:
rather than have extraArgs could you rename this to `.Values.container.args` and set the default values to 
``` 
- --collector.procfs=/host/proc
- --collector.sysfs=/host/sys
```

Reply:
ok. it would make sense to do the same with volumes and mounts then

Reply:
Indeed!